### DISCUSSION TITLE
Ajouter une catégorie sur Mes Achats

### DISCUSSION THREAD

**Discussion by ghost** [betagouv/ma-cantine#1232] [Idées]
Une utilisatrice (L. A.) m'a fait remarquer qu'elle ne sait pas dans quelle catégorie ranger ses factures de frites surgelées et celles de pommes de terre sous vide sur la page Mes Achats.
=> Suggestion : ajouter une catégorie "Féculents" ou "Riz, pâtes, pommes de terre" dans Mes Achats

**Comment by ghost** [comment]
@JenniferStephan, @valeriemerle, @alemangui & @hfroot, je profite de l'atelier Brainstorming pour relancer cette demande d'utilisatrice.

**Comment by hfroot** [comment]
On est en train de remplacer les catégories avec des familles : https://github.com/betagouv/ma-cantine/issues/1511 - est-ce que ça clarifie quoi à choisir dans ce cas ?

**Reply by ghost** [reply]
Je pense que ça règle le pb en effet : elle rangerait ses frites surgelées dans la famille "Autres produits frais, surgelés et d’épicerie ☃️". Merci, @hfroot.
